bigtable/noxfile.py

```diff
@@ -23,40 +23,36 @@
 LOCAL_DEPS = (os.path.join("..", "api_core"), os.path.join("..", "core"))
 
 @nox.session(python="3.7")
-def blacken(session):
-    """Run black.
+def lint(session):
+    """Run linters.
 
-    Format code to uniform standard.
+    Returns a failure if the linters find linting errors or sufficiently
+    serious code quality issues.
     """
-    session.install("black")
+    session.install("flake8", "black", *LOCAL_DEPS)
     session.run(
         "black",
+        "--check",
         "google",
         "tests",
         "docs",
-        "--exclude",
-        ".*/proto/.*|.*/gapic/.*|.*/.*_pb2.py",
     )
+    session.run("flake8", "google", "tests")
 
 
-@nox.session(python="3.7")
-def lint(session):
-    """Run linters.
+@nox.session(python="3.6")
+def blacken(session):
+    """Run black.
 
-    Returns a failure if the linters find linting errors or sufficiently
-    serious code quality issues.
+    Format code to uniform standard.
     """
-    session.install("flake8", "black", *LOCAL_DEPS)
+    session.install("black")
     session.run(
         "black",
-        "--check",
         "google",
         "tests",
         "docs",
-        "--exclude",
-        ".*/proto/.*|.*/gapic/.*|.*/.*_pb2.py",
     )
-    session.run("flake8", "google", "tests")
```

Comment:
@crwilcox Why would we want to run `lint` before `blacken`?

Reply:
This PR got lost in the pile. The reason is so CI fails if un-blackened code is pushed. If we do it the other way lint simply validates blackening, not the state of the commitish under test.